apps/plugin-browser/package.json

```diff
@@ -32,6 +33,7 @@
     "@local/hash-isomorphic-utils": "0.0.0-private",
     "@local/hash-subgraph": "0.0.0-private",
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
+    "@sentry/webpack-plugin": "1.20.0",
```

Comment:
With `1.20.0` I was getting a `sentryWebpackPlugin is not a function` error, I was able to fix this by upgrading to `2.9.0`.